fmriprep/workflows/bold.py

```diff
@@ -1735,3 +1776,19 @@ def init_func_derivatives_wf(output_dir, output_spaces, template, freesurfer,
         ])
 
     return workflow
+
+
+def _get_series_len(bold_fname):
+    import nibabel as nb
+    from niworkflows.nipype.algorithms.confounds import is_outlier
+    img = nb.load(bold_fname)
+    if len(img.shape) < 4:
+        return 1
+
+    # Replicate work done in EstimateReferenceImage to have access to
+    # skip_vols at workflow creation time
+    data_slice = img.dataobj[:, :, :, :50]
```

Comment:
What is the behavior if `img.dataobj.shape[3] < 50` here? Would this break?

Reply:
No, it will be equivalent to `img.dataobj[:, :, :, :]`.